io.kapsa.drive.json

```diff
@@ -0,0 +1,365 @@
+{
+  "app-id": "io.kapsa.drive",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
```

Comment:
current runtime is now 23.08

Reply:
We've tried using `23.08`, but we can't manage to compile the `ffmpeg` dependency due to: `./libavcodec/x86/mathops.h:125: Error: operand type mismatch for `shr'` likely related: https://www.linuxquestions.org/questions/slackware-14/regression-on-current-with-ffmpeg-4175727691/

Once we've solved this issue and potentially any others that may arise after upgrading to fresh 23.08 we would be using 22.08.

Reply:
Hi @barthalion,
I've noticed that you've changed the `runtime-version` to `23.08`: https://github.com/s3drive/flathub/commit/57c087867ffedd821cc934fca771247b957786b4
I've failed so far to compile dependencies, we're having issue at least with `ffmpeg`, can we keep it: `22.08` and aim to update to `23.08` over the next couple months?

https://github.com/flathub/flathub/pull/4364#discussion_r1321336267

Reply:
There is that upstream patch you can find by Googlling:

https://github.com/flathub/org.ardour.Ardour/blob/master/patches/ffmpeg-as-fix.patch

Reply:
Thanks for linking the fix. I've applied it and it resolved the `ffmpeg` compile issue at least, but now I am getting: 
```
ar: error while loading shared libraries: libsframe.so.1: cannot open shared object file: No such file or directory
```
during installation.
It seems it fails on: `ar x s3drive_amd64.deb` however when I shell into:
```
flatpak-builder --build-shell=s3drive build-dir io.kapsa.drive.json
```
and run the: `ar x s3drive_amd64.deb` after fetching the .deb first, it seem to work.

These issues does not appear on: `22.08`.